app/main/appium-method-handler.js

```diff
@@ -242,9 +251,15 @@ export default class AppiumMethodHandler {
        */
       try {
         if (currentContext !== NATIVE_APP) {
-          const webviewStatusAddressBarHeight = await this.driver.execute(getWebviewStatusAddressBarHeight, [{platformName, statBarHeight}]);
-
-          await this.driver.execute(setHtmlElementAttributes, [{platformName, webviewStatusAddressBarHeight}]);
+          // Fallback if the webview position can't be determined,
+          // then do it based on the web context
+          if (!webViewPosition) {
+            webViewPosition = {
+              x: 0,
+              y: await this.driver.execute(getWebviewStatusAddressBarHeight, [{platformName, statBarHeight}]),
+            };
+          }
+          await this.driver.execute(setHtmlElementAttributes, [{platformName, webviewStatusAddressBarHeight: webViewPosition.y}]);
```

Comment:
previously we were passing `statBarHeight`. Is it expected that we change its name to `webviewStatusAddressBarHeight`?

Reply:
Hi @mykola-mokhnach 

Not sure what you mean, the arguments are exactly the same, only an extra if else check

Reply:
sorry, got it now. Please ignore